kclvm/tools/src/LSP/src/config_manager.rs

```diff
@@ -0,0 +1,56 @@
+use serde::{Deserialize, Serialize};
+use std::path::PathBuf;
+
+#[derive(Serialize, Deserialize, Clone)]
+struct JsonFile {
+    watch: PathBuf,
+    recursive: Option<bool>,
+    patterns: Option<Vec<String>>,
+}
+
+#[derive(Debug, Clone)]
+pub struct Config {
+    path: PathBuf,
+    recursive: bool,
+    patterns: Vec<String>,
+}
+
+impl Config {
+    // Load configuration from file
```

Comment:
You can use doc comment
```suggestion
    /// Load configuration from file
```